CMakeLists.txt

```diff
@@ -195,6 +195,24 @@ if(WITH_GSL)
   endif(GSL_FOUND)
 endif(WITH_GSL)
 
+find_package(BLAS)
+if(BLAS_FOUND)
+  set(BLAS 1)
+endif()
+find_package(LAPACK)
+if(LAPACK_FOUND)
+  set(LAPACK 1)
+endif()
+
+if(BLAS AND LAPACK)
+  set(STOKESIAN_DYNAMICS 1)
+endif()
```

Comment:
why don't you directly use ```LAPACK_FOUND``` and ```BLAS_FOUND```?

Reply:
I tried to use the line `if(BLAS_FOUND AND LAPACK_FOUND)` and it didn't work as expected, at least on the Clang images. Would that be the correct way to do it?

Reply:
we need to declare `BLAS` and `LAPACK` variables in CMake, so that we can export them in a .hpp file using a python script that looks for the following keywords:
https://github.com/espressomd/espresso/blob/b88c138580cfcad5aa51da977271f45d6266ab34/src/config/features.def#L116-L117
Commit dc61284ba879 removed these variables. It would have worked by just replacing `if(BLAS AND LAPACK)` by `if(BLAS_FOUND AND LAPACK_FOUND)`. I don't have a strong opinion on using `FEATURE` vs `FEATURE_FOUND` in this case, though. This distinction is relevant if a library X is required to enable features Y and Z, but Y and Z are independent, in which case `if(X_FOUND)` should be used instead of `if(Y)` (assuming X is not a feature). However, the way `features.def` is currently written, `BLAS` and `LAPACK` are both libraries and features.

Reply:
Ok, since the variables `BLAS` and `LAPACK` need to be declared anyway, I will leave it as it was before commit dc61284 (reverted the commit already). Thanks.